django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from django.conf import settings
+from django.db import models
+
+
+class Comment(models.Model):
+    package = models.ForeignKey(
```

Comment:
How about we make this a GenericForeignKey instead, allowing the attachment of comments to any arbitrary model?